[2025-07-28T15:02:24.080+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: orders_dag.fetch_and_load_orders manual__2025-07-28T15:02:21.523672+00:00 [queued]>
[2025-07-28T15:02:24.090+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: orders_dag.fetch_and_load_orders manual__2025-07-28T15:02:21.523672+00:00 [queued]>
[2025-07-28T15:02:24.090+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-07-28T15:02:24.105+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): fetch_and_load_orders> on 2025-07-28 15:02:21.523672+00:00
[2025-07-28T15:02:24.116+0000] {standard_task_runner.py:57} INFO - Started process 26476 to run task
[2025-07-28T15:02:24.123+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'orders_dag', 'fetch_and_load_orders', 'manual__2025-07-28T15:02:21.523672+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/orders_day.py', '--cfg-path', '/tmp/tmpgj5bfdzr']
[2025-07-28T15:02:24.127+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask fetch_and_load_orders
[2025-07-28T15:02:24.283+0000] {task_command.py:416} INFO - Running <TaskInstance: orders_dag.fetch_and_load_orders manual__2025-07-28T15:02:21.523672+00:00 [running]> on host 979e7d3a451f
[2025-07-28T15:02:24.388+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='orders_dag' AIRFLOW_CTX_TASK_ID='fetch_and_load_orders' AIRFLOW_CTX_EXECUTION_DATE='2025-07-28T15:02:21.523672+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-28T15:02:21.523672+00:00'
[2025-07-28T15:02:24.391+0000] {logging_mixin.py:151} INFO - ðŸš€ Starting data fetch from API...
[2025-07-28T15:02:24.476+0000] {logging_mixin.py:151} INFO - ðŸ”Ž Status Code: 200
[2025-07-28T15:02:24.476+0000] {logging_mixin.py:151} INFO - ðŸ”Ž Raw Response: [{"order_id":"00125cb692d04887809806618a2a145f","user_name":"7ac26eac431c6848694a2de6f0327524","order_status":"delivered","order_date":"3/23/2017 12:21","order_approved_date":"3/23/2017 13:05","pickup...
[2025-07-28T15:02:24.479+0000] {logging_mixin.py:151} INFO - âœ… Successfully parsed JSON. Total Orders: 602
[2025-07-28T15:02:24.486+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/orders_day.py", line 37, in fetch_and_load_orders
    cursor.execute("""
psycopg2.errors.InvalidTextRepresentation: invalid input syntax for type integer: "00125cb692d04887809806618a2a145f"
LINE 3:             VALUES ('00125cb692d04887809806618a2a145f', NULL...
                            ^

[2025-07-28T15:02:24.500+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=orders_dag, task_id=fetch_and_load_orders, execution_date=20250728T150221, start_date=20250728T150224, end_date=20250728T150224
[2025-07-28T15:02:24.517+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 14 for task fetch_and_load_orders (invalid input syntax for type integer: "00125cb692d04887809806618a2a145f"
LINE 3:             VALUES ('00125cb692d04887809806618a2a145f', NULL...
                            ^
; 26476)
[2025-07-28T15:02:24.541+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-07-28T15:02:24.573+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
